Source/Fuse.Share/ShareModule.uno

```diff
@@ -80,6 +104,22 @@ namespace Fuse.Share
 			return false;
 		}
 
+		bool TryGetPosition(object arg, out float2 position)
+		{
+			position = float2(0.0f);
+			var obj = arg as Fuse.Scripting.Object;
+			if (obj != null && obj.ContainsKey("external_object"))
+			{
+				var element = ((Fuse.Scripting.External)obj["external_object"]).Object as Element;
+				if (element != null)
+				{
+					position = element.ActualPosition;
```

Comment:
Doesn't this need to be translated into screen coordinates, or relative to the NativeView?

`ActualPosition` is not likely what we want here, either the center of the element or `ActualAnchor` would make more sense (as a default at least).